models/classes/CrudResultsService.php

```diff
@@ -15,34 +15,33 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- * Copyright (c) 2017 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ * Copyright (c) 2017-2020 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
  *
  */
 
 namespace oat\taoResultServer\models\classes;
 
-use common_exception_Error;
 use core_kernel_classes_Class;
-use oat\taoDelivery\model\execution\ServiceProxy;
 use oat\taoDeliveryRdf\model\DeliveryAssemblyService;
+use oat\taoResultServer\models\Formatter\ItemResponseCollectionNormalizer;
+use oat\taoResultServer\models\Formatter\ItemResponseVariableSplitter;
+use tao_models_classes_CrudService;
 use taoResultServer_models_classes_ReadableResultStorage;
 use taoResultServer_models_classes_ResponseVariable;
 
 /**
- * .Crud services implements basic CRUD services, orginally intended for REST controllers/ HTTP exception handlers
- *  Consequently the signatures and behaviors is closer to REST and throwing HTTP like exceptions
- *
- *
- *
+ * Crud services implements basic CRUD services, originally intended for REST controllers/ HTTP exception handlers
+ * Consequently the signatures and behaviors is closer to REST and throwing HTTP like exceptions
```

Comment:
Not related to your development, just a highlight. It makes me think. Service is doing a "format" what is not related to CRUD. Comments seems not necessary and as you see it gets outdated and starts telling lies.